_posts/2020-04-15-cockpit-217.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 217
+author: mvo
+date: 2020-04-15
+tags: cockpit linux
+slug: cockpit-217
+category: release
+summary: Cockpit with improved ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 217.
+
+### More details about Insights rule hits
```

Comment:
Please prefix with the page, "Overview:".

Reply:
Done.